src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -12,20 +12,27 @@
     <IncludeSymbols>false</IncludeSymbols>
     <DebugType>embedded</DebugType>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Sql</PackageId>
+    <RootNamespace></RootNamespace>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
what's this newfolder stuff?

Reply:
hah, oops. Was messing around with stuff and apparently VS didn't clean this up as nicely as I thought

Reply:
just saw the property above too. coul dyou take a look?

Reply:
Yeah the root namespace? I added a comment explaining that. I think it's a bug with the analyzer or something...for files in the root of the project folder it always gives an error about the namespace.

I'll probably restructure them to be in sub-folders (which fixes that issue) in the future but for now this seems fine enough.